src/styles/blocks/header.scss

```diff
@@ -0,0 +1,56 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px 0 #0000000d;
+
+  &__logo {
+    display: block;
+    width: 40px;
+    height: 40px;
+  }
+}
+
+.nav {
+  &__list {
+    display: flex;
+    list-style: none;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__link {
+    display: block;
+    position: relative;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: $header-line-height;
+    text-transform: uppercase;
+    text-decoration: none;
+    color: $text-color-navy-blue;
+    text-align: center;
+
+    &:hover {
+      color: $color-vivid-cerulean;
+      transition: all 0.3s;
```

Comment:
The transition property should be placed outside of the pseudo-class to ensure it applies to all state changes, not just the hover state. Consider moving `transition: all 0.3s;` to the `&__link` block.